### PR TITLE
Limit NEST 5.x compatibility to Elasticsearch.Net 5.x versions

### PR DESCRIPTION
**Pull request by russcam**

### CODE REVIEW

**Comment by Mpdreamz** [comment]
LGTM is there anyway we can make the build smarter here as well so we do not need to bump on each major version?

**Comment by russcam** [comment]
I have a fix for this

**Comment by russcam** [comment]
I've updated the commit to pass the next major version and current year to nuget pack.

File: build/scripts/Releasing.fsx [hunk 1]
```diff
@@ -23,7 +30,33 @@ module Release =
             let nuspec = (sprintf @"build\%s.nuspec" name)
             let nugetOutFile =  Paths.Output(sprintf "%s.%s.nupkg" name Versioning.FileVersion)
 
-            Tooling.Nuget.Exec ["pack"; nuspec; "-version"; Versioning.FileVersion; "-outputdirectory"; Paths.BuildOutput; ] |> ignore
+            let nextMajorVersion =
```
Comment (Mpdreamz) [inline build/scripts/Releasing.fsx]: 👍 very nice!

**Comment by Mpdreamz** [comment]
ported to `master`

**Pull request merged**